portfolio/src/main/webapp/contact.html

```diff
@@ -31,7 +31,17 @@ <h2>You can also find me on LinkedIn and Github.</h2>
     <br>
     <div class="content comments">
         <h2>Sign the Guestbook!</h2>
-        <form action="/data" method="POST">
+        <form action="/comments" method="POST">
+            <div id="toggle-comments">
+                <label for=:"comment-limit">View</label>
+                <select name="comment-limit" id="comment-limit" onchange="getComments()">
+                    <option value="5">5</option>
+                    <option value="10">10</option>
+                    <option value="all" selected="selected">All</option>
```

Comment:
This is where you could set the value to be "-1".

This is a bit of a trade-off, in that it does sort of expose internal details of how the system operates. So an argument could be made that handling "all" specially in the backend is in some sense cleaner. So you should feel free to make that argument.

There are times when it makes sense to disagree with review comments. This can be one of them.
If you disagree, then you should explain why you think a different choice makes sense.

Reply:
Hmm. I don't think exposing that internal detail would be that much of a problem here, so I will go ahead with replacing the "all"